web/crux-ui/src/components/products/versions/images/image-config-filters.tsx

```diff
@@ -0,0 +1,149 @@
+import { DyoLabel } from '@app/elements/dyo-label'
+import { ImageConfigFilterType, IMAGE_CONFIG_FILTERS } from '@app/models'
+import clsx from 'clsx'
+import useTranslation from 'next-translate/useTranslation'
+import { useEffect, useState } from 'react'
+
+type BaseImageConfigFilterType = 'all' | 'common' | 'docker' | 'kubernetes'
+type FilterSet = Record<BaseImageConfigFilterType, ImageConfigFilterType[]>
+
+export const filterSet: FilterSet = {
+  all: [],
+  common: [
+    'name',
+    'environments',
+    'secrets',
+    'ingress',
+    'expose',
+    'user',
+    'tty',
+    'importContainer',
+    'configContainer',
+    'ports',
+    'portRanges',
+    'volumes',
+    'commands',
+    'args',
+    'initContainers',
+  ],
+  kubernetes: [
```

Comment:
In proto we use Dagent and Crane, I just think this is a good approach to use here in a different way. A different abstraction layer, I know but it will be misunderstood in a long run, IMHO. What do you think?